packages/firebase_auth/lib/firebase_auth.dart

```diff
@@ -20,9 +20,9 @@ class FirebaseUserMetadata {
 }
 
 class UserInfo {
-  UserInfo._(this._data, this.app);
+  UserInfo._(this._data, this._app);
 
-  FirebaseApp app;
+  final FirebaseApp _app;
```

Comment:
Since this version with FirebaseApp was not published yet, this is not a breaking change.